AVAL-Spec.yml

```diff
@@ -491,13 +491,13 @@ definitions:
         $ref: "#/definitions/Period"
       requiredDocuments:
         type: "array"
-        description: "document types as Defined in AvalDocuments/fileType"
+        description: "document types as Defined in AvalDocuments/fileType; those documents must be made available before the transaction state goes to 9"
```

Comment:
The spec names the attribute which describes the document type as _"documentType"_ and not _"fileType"_. See  _"/avalmatchings/{avalId}/avaltransactions/{transactionId}/avaldocuments"_ parameters section for more details.

It would be better to use the same naming here and in line 500 too.

PS: _**D**_efined -> _**d**_efined.

Reply:
yes, I think this was renamed in the avalDocuments after the initial creation of aval2matching and then we did miss this change. btw. the pdf documentation with the curl sample for documents is partially broken regarding those field names as well.